.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--- indicate the Title for this pull request (PR) above -->
+
+
+Thank you for considering to contribute to the INBO tutorials repository.
```

Comment:
As the PR is generally added after the contribution has already been made, I would remove the 'considering to'.  It somehow gives the feeling that it is not sure yet if the contribution will be added.

```suggestion
Thank you for contributing to the INBO tutorials repository.
```